core/src/test/scala/com/microsoft/ml/spark/nbtest/DatabricksUtilities.scala

```diff
@@ -50,7 +50,9 @@ object DatabricksUtilities extends HasHttpClient {
     Map("pypi" -> Map("package" -> "nltk")),
     Map("pypi" -> Map("package" -> "bs4")),
     Map("pypi" -> Map("package" -> "plotly")),
-    Map("pypi" -> Map("package" -> "Pillow"))
+    Map("pypi" -> Map("package" -> "Pillow")),
+    Map("pypi" -> Map("package" -> "onnxmltools")),
+    Map("pypi" -> Map("package" -> "lightgbm"))
```

Comment:
Are we able to switch this back to using our LGBM?

Reply:
Unfortunately no. Our LGBM outputs the string representation for a "Booster" model, and I cannot find an API to convert it to a "Classifier" model. `onnxmltools` (as well as Hummingbird) only supports conversion of a "Classifier" model, but not a "Booster" model.

Reply:
@memoryz you can use loadNativeModelFromString or loadNativeModelFromFile to load a native lightgbm model file to create an mmlspark LightGBMClassificationModel:

https://github.com/Azure/mmlspark/blob/master/lightgbm/src/main/python/mmlspark/lightgbm/LightGBMClassificationModel.py#L25

Reply:
> @memoryz you can use loadNativeModelFromString or loadNativeModelFromFile to load a native lightgbm model file to create an mmlspark LightGBMClassificationModel:
> 
> https://github.com/Azure/mmlspark/blob/master/lightgbm/src/main/python/mmlspark/lightgbm/LightGBMClassificationModel.py#L25

But what I need is a non-mmlspark `LGBMClassifier`: https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.LGBMClassifier.html#lightgbm.LGBMClassifier.

What I'm doing here is to use onnxmltools to convert it to a ONNX model. I don't think onnxmltools works with MMLSpark's `LightGBMClassificationModel` class.

Reply:
Let me look into this a bit more.

Reply:
so you mean that you want to create a lightgbm model, specifically LGBMClassifier, from a native lightgbm file?
Interesting, you can unofficially do it like this:
https://github.com/interpretml/interpret-community/blob/0923ad59f104f788941f39a197387748f8bfbef4/python/interpret_community/mimic/models/lightgbm_model.py#L397


see issues for reference:
https://github.com/Microsoft/LightGBM/issues/1217
https://github.com/Microsoft/LightGBM/issues/1942

I'm not sure how that relates to the mmlspark lightgbm model that I thought Mark was referring to.

Reply:
> @memoryz so you mean that you want to create a lightgbm model, specifically LGBMClassifier, from a native lightgbm file?
> Interesting, you can unofficially do it like this:
> https://github.com/interpretml/interpret-community/blob/0923ad59f104f788941f39a197387748f8bfbef4/python/interpret_community/mimic/models/lightgbm_model.py#L397
> 
> see issues for reference:
> [microsoft/LightGBM#1217](https://github.com/microsoft/LightGBM/issues/1217)
> [microsoft/LightGBM#1942](https://github.com/microsoft/LightGBM/issues/1942)
> 
> I'm not sure how that relates to the mmlspark lightgbm model that I thought Mark was referring to.

I'll take a look and get back to this.

Reply:
Turns out the onnxmltools package does support converting a lightgbm booster model directly. It's Hummingbird that doesn't support the conversion of booster. Since we're not using Hummingbird for now, I modified the notebook to use mmlspark's LightGBM to train the model, then export the a booster, then convert to onnx. We still need the lightgbm python package though, because the booster object is defined there.